ui/material.py

```diff
@@ -107,6 +107,26 @@ def draw(self, context):
                 op.preset = preset
 
 
+class LUXCORE_PT_settings(MaterialButtonsPanel, Panel):
+    bl_label = "Settings"
+    bl_context = "material"
+    bl_options = {'DEFAULT_CLOSED'}
+
+    @classmethod
+    def poll(cls, context):
+        engine = context.scene.render.engine
+        return engine == "LUXCORE"
+
+    def draw(self, context):
+        layout = self.layout
+        mat = context.material
+        core = mat.luxcore
+
+        split = layout.split()
+        col = split.column()
+        layout.prop(mat, "diffuse_color", text="Viewport Color")
+        layout.prop(core, "auto_mat_selection", text="Automatic Material Selection:")
```

Comment:
```suggestion
        layout.prop(core, "auto_mat_selection")
```
Remove the `text` argument to prevent code duplication (the `name` of the property will be used by Blender).